lib/src/signed_video_internal.h

```diff
@@ -105,6 +105,8 @@ struct _sei_data_t {
   uint8_t *payload;  // Pointer to the allocated SEI data
   uint8_t *payload_signature_ptr;
   uint16_t last_two_bytes;
+  uint8_t *completed_sei;  // Number of completed sei
```

Comment:
Remove this member. It is never used.
In another PR we should rename "payload" to "sei" and "payload_signature_ptr" to "write_position"